lightly/models/examples/moco_system.py

```diff
@@ -0,0 +1,67 @@
+import pytorch_lightning as pl
+import torch
+
+from lightly.models.modules.heads import MoCoProjectionHead
+
+import lightly
+from lightly.models.modules.momentum_encoder import MomentumEncoder
+
+
+class MocoSystem(pl.LightningModule):
+    def __init__(self, dataloader_kNN, num_classes):
```

Comment:
The `dataloader_kNN` and `num_classes` are just a copy-paste artefact, right?

Reply:
Yes, sorry